src/Simplex/Chat.hs

```diff
@@ -3602,7 +3612,13 @@ processAgentMessageConn user@User {userId} corrId agentConnId agentMessage = do
             | sameMemberId memberId mem && msgMemberId == memberId -> delete ci Nothing >>= toView
             | otherwise -> deleteMsg mem ci
           CIGroupSnd -> deleteMsg membership ci
-        Left e -> messageError $ "x.msg.del: message not found, " <> tshow e
+        Left e
+          | msgMemberId /= memberId ->
+            withStore' (\db -> runExceptT $ getGroupMemberByMemberId db user groupId msgMemberId) >>= \case
+              Right itemMember -> checkRole itemMember $
+                withStore' $ \db -> createChatItemModeration db gInfo m msgMemberId sharedMsgId msgId brokerTs
+              Left e' -> messageError $ "x.msg.del: message sender not found, " <> tshow e'
```

Comment:
I think it might be possible that member doesn't exist - I think it should be applied at the point of content being received, so you won't need getGroupMemberByMemberId

Reply:
with reactions, if the item is not found the reaction is just saved, nothing happens with the member. I think it should be the same here, as you may not see the member yet, but would see them later.